ovos_PHAL_plugin_system/__init__.py

```diff
@@ -37,11 +41,20 @@ def handle_reboot_request(self, message):
 
     def handle_shutdown_request(self, message):
         system_shutdown()
+        
+    def handle_configure_language_request(self, message):
+        language_code = message.data.get('language_code', "en_US")
+        bash_profile_file = open(f"{os.environ['HOME']}/.bash_profile", "w")
+        bash_profile_file.write(f"export LANG={language_code}\n")
+        bash_profile_file.close()
+        set_default_lang(language_code)
```

Comment:
the lang code here should be normalized, replace `_` with ` -` which is expected by core everywhere